align.js

```diff
@@ -3,18 +3,23 @@ var tact = require('./tact/tact.js');
 
 var cli = require('cli');
 var options = cli.parse({
+  sourceLanguage: ['x', 'source language name', 'string', ''],
+  targetLanguage: ['y', 'target language name', 'string', ''],
   sourceFile: ['s', 'source file to be aligned', 'file', ''],
   targetFile: ['t', 'target file to be aligned', 'file', ''],
   first: ['f', 'first input line to use', 'int', 0],
   count: ['n', 'limit input to n lines', 'int', 20]
 });
 
+tact.config.global.sourceLanguage = options.sourceLanguage
+tact.config.global.targetLanguage = options.targetLanguage
```

Comment:
yeah, I think it passes the empty string. I removed those in the latest push, and will worry about them once I can test it.